smriprep/workflows/anatomical.py

```diff
@@ -972,6 +973,9 @@ def init_anat_fit_wf(
     templates = []
     found_xfms = {}
     for template in spaces.get_spaces(nonstandard=False, dim=(3,)):
+        from smriprep.interfaces.templateflow import fetch_template_files
+
+        fetch_template_files(template, specs=None, sloppy=sloppy)
```

Comment:
This feels like the wrong place to put this. Could we instead put it in the workflows that use `TemplateFlowSelect`, to more tightly tie the places it's needed?

Reply:
where would you suggest we put it? The reason I chose this spot is it is hit regardless of whether derivatives are found, and didn't want to duplicate this call across workflows.

Reply:
Well I was thinking that it's okay to duplicate, to make sure that it's done however workflows are mixed and matched.

Reply:
[e890872](https://github.com/nipreps/smriprep/pull/418/commits/e890872e4594ac590546dba23b1e9bc362969a3a) moves this into `init_template_iterator_wf`